.travis.yml

```diff
@@ -12,19 +12,15 @@ install:
   - pip install --upgrade --pre tox
 env:
   matrix:
-    # note: please use "tox --listenvs" to populate the build matrix below
-    # please remove the linting env in all cases
-    - TOXENV=py27-pexpect
+    # Specialized factors for py27.
```

Comment:
What was the reasoning here for plumping pexpect, trial, and numpy together, while keeping xdist and pluggymaster separate? I'm curious.

Reply:
xdist and pluggymaster run the whole test suite, while the others only run parts of it.

Initially I've thought to just install pexpect on CI by default, but then it would slow more jobs down.

Re pexpect I think it could even use a marker or similar to really run only pexpect tests maybe even.

Reply:
> Please add a comment so the rationale is explicit.

"Specialized factors for py27." was meant to be for that.  But maybe put it on the same line then?  Better wording?

Reply:
"Specialized tests for py27"? Not sure, the wording might be fine and it was my fault for not understanding it right away hehehe